web/forms.py

```
@@ -1,4 +1,5 @@
 import re
+from typing import Any, ClassVar, Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)